packages/palette/src/svgs/BriefcaseIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import { color } from "../helpers"
+import { Icon, IconProps, Path, Title } from "./Icon"
+
+/** BriefcaseIcon */
+export const BriefcaseIcon: React.SFC<IconProps> = ({
+  title = "Briefcase",
```

Comment:
I'm curious if there's a more semantic default title for this icon. In general I think we try to use titles that [tell the user what the icon is going to do for them rather than what it looks like](https://github.com/artsy/palette/blob/master/packages/palette/src/svgs/EnvelopeIcon.tsx#L7). My understanding is that these titles would be used by a screenreader, so something like "my documents" (or something....) would give the user more context than "briefcase".

Reply:
(Though, that's just the default prop — so it should just be set to whatever in the direct context)

Reply:
Understood & agreed, but the likelihood of people overriding the title on an icon seems very low.

Reply:
Ah great call out here, thanks for raising. Happy to open up a patch to address this, we will be using this to display a collectors profession in the collector profile experience so maybe `"profession"` is a better fit here

Reply:
> but the likelihood of people overriding the title on an icon seems very low

ha: remove the default and make it a required prop for every icon. TBH probably much better from an accessibility standpoint. 🤷

Reply:
@pepopowitz you are very right — I'm doing an a11y pass on the main nav and our notifications icon is labelled 'Watch lot' 😞 

![](http://static.damonzucconi.com/_capture/SdzoQ8prvwNk.png)